TriblerGUI/widgets/trustpage/js/drawing.js

```diff
@@ -47,21 +48,31 @@ function drawNodes(svg, data, on_click) {
         .attr("fill", function (d) {
             return getNodeColor(d, data)
         })
-        .attr("r", "0")
+        .attr("r", function (d) {
+            return getNodeRadius(d, data)
+        })
```

Comment:
Please remove this piece, the new nodes must start at radius 0. On line 74 you set their final value after a transition of 1000 ms.